README.md

```diff
@@ -1,2 +1,64 @@
 # Resturant-Page
 A simple web app to dislay a resturant page using HTML, JavaScript and Webpack.
+
+
+Things covered in the project:
+
+- Factory Functions
+- Module pattern
+- Modules
+- Webpack
+
+
+
+## Build with
+
+- JavaScript
+- Webpack
+- CSS3
+
+
+## How to run this app
+To run it, clone this repository locally, and run these commands on your terminal.
+```
+Git clone git@github.com:Sick-Carlito/Resturant-Page.git
+
+cd ~/Resturant-Page
+
+```
+
+Note: If you don't have it yet, you need to install webpack
+in your root directory and run the following command.
+```
+npx webpack
+```
+
+Now, open the index.html file in your browser.
```

Comment:
- [ ] is `npm i` not necessary? What about the dependencies at your package.json?
Make the instructions are complete.